Libplanet/Blocks/BlockHeader.cs

```diff
@@ -200,6 +211,14 @@ internal void Validate(DateTimeOffset currentTime)
                 );
             }
 
+            if (Difficulty > TotalDifficulty)
+            {
+                throw new InvalidBlockTotalDifficultyException(
+                    "total difficulty must be less than its difficulty, " +
```

Comment:
Shouldn't it be the other way?

```suggestion
                    "Block difficulty must be less than its total difficulty, " +
```

Reply:
How about this:

```
A Block.Difficulty ({Difficulty}) must be less than its TotalDifficulty ({TotalDifficulty}).
```

Reply:
Other exception messages used to start with small letter. Use capital anyways?

Reply:
Microsoft's exception messages seem to tend to start with capital letters.  🤔  Anyway, we've used to start with capital letters too, in other places.